test/evergreen.yml

```diff
@@ -3703,6 +3703,20 @@ tasks:
             g++ -o memory_model_test -O2 memory_model_test.cpp -lpthread -std=c++17 -Wall -Werror -DAVOID_CPP20_SEMAPHORE
             ./memory_model_test -n 100000000
 
+  - name: tsan-playground-test
```

Comment:
Is there benefit to run TSAN playground on Evergreen file? IMO, if it doesn't assist testing we shouldn't add it

Reply:
We already have `memory-model-test` for in the evergreen for testing a similar tool. And we run it once a month just to make sure that everything is compiling and we keep our repo outdated.

If we don't want to care about having it up-to-date, I think we shouldn't put this tool to the WT public repo.